src/mixins/toolpop.js

```diff
@@ -59,12 +59,6 @@ export default {
       type: String,
       default: null
     },
-    boundary: {
-      // String: scrollParent, window, or viewport
-      // Element: element reference
-      type: [String, HTMLElement],
-      default: 'scrollParent'
-    },
     boundaryPadding: {
```

Comment:
Just thinking, since `boundaryPadding` is also related to `boundary`, it should probably be moved into the config as well.

Reply:
I don't  see the boundaryPadding in the Bootstrap documentation, and it event isn`t to much information about it in bootstrap-vue.

I try to use it but didn`t see the effect.

I believe since, its not a oficial bootstrap config option, we could leave it as it is.

I try to find, but didn`t understand the use case for it.

Reply:
The boundary padding option makes sure the toolti/popover is inside the boundary element by # pixels.  say if the boundary element was window, and boundary padding was set to 0, then the tooltip could touch the edge of the window.  The current default is 5 (5px), so it will try to stay away from edge of the boundary element by 5px.

Reply:
It is a popper config option (which you can pass via the config normally) bootstrap v4 provides a link to Popper options in their docs.

Reply:
I've added in the boundaryPadding to the config.